packages/mantine/src/theme/plasmaCSSVariablesResolver.ts

```diff
@@ -56,7 +56,7 @@ export const plasmaCSSVariablesResolver: CSSVariablesResolver = (theme) => {
             '--coveo-color-bg-dark-surface': theme.colors.violet[9],
 
             // mantine overrides
-            '--mantine-color-body': theme.colors.gray[0],
+            '--mantine-color-body': 'var(--mantine-color-white)',
```

Comment:
Should we change the dark one as well so color-body and app-background are different?